### DISCUSSION TITLE
next-intl does not work on dev --turbo

### DISCUSSION THREAD

**Discussion by MirroxDev** [amannn/next-intl#921] [Q&A]
Hello everyone, please tell me, I use next-intl and when I run “next dev” intl works fine, but “next dev --turbo” no longer works, the language simply does not change, and there are no errors. Is this a bug or did I screw up somewhere?
 
"next": "^14.1.1",
"next-intl": "^3.9.1",

**Comment by AbrahamX3** [comment]
Oh, thanks for this, I was going crazy figuring out why it wasn't working, It kept using any other locale instead of the default locale, running the dev server without turbo fixed it for me, hopefully this gets fixed soon
